examples/profile.php

```diff
@@ -17,7 +17,9 @@
     $base64Selfie = ActivityDetailsHelper::getBase64Selfie($profile);
     // This key uses the  format: age_[over|under]:[1-999] and is dynamically
     // generated based on the dashboard attribute Age / Verify Condition
-    $ageVerified = $profile->getProfileAttribute('age_over:18');
+    $ageVerified = $profile->isAgeVerified() ? 'true' : '';
```

Comment:
What does this do? Are you converting it to a string or not wanting to display `false`?